src/api/cvc4cpp.h

```diff
@@ -1218,9 +1218,9 @@ class CVC4_PUBLIC DatatypeSelector
 
   /**
    * Get the selector operator of this datatype selector.
-   * @return the selector operator
+   * @return the selector Term
```

Comment:
Term -> term

Reply:
Good catch, thanks! Fixed in 8777f854d5b024519116300a8ee0ac45411507ad.